platforms/documentation/docs/src/docs/userguide/dep-man/centralizing-dependencies/centralizing_catalog_platform.adoc

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[platforms-vs-catalog]]
+= Using Catalogs with Platforms
+
+Both platforms and version catalogs help manage dependency versions in a project, but they serve different purposes and have different effects on dependency resolution:
+
+== Version Catalogs
+
+- **Purpose:** A version catalog is used to centralize and standardize dependency coordinates (group, name, version). It provides type-safe accessors in the build script, making dependencies easier to manage.
+- **Effect on Dependency Graph:** Version catalogs **do not** affect dependency resolution. The versions defined in the catalog are **suggestions** and **transparent to downstream consumers**. These versions can be overridden by other mechanisms like transitive dependencies or platforms.
+
+[source,toml]
+.libs.versions.toml
+----
+[libraries]
+mylib = { group = "com.example", name = "mylib", version = "1.0.0" }
+----
+
+== Platforms
+
+- **Purpose:** A platform is a module in the dependency graph that enforces or aligns versions of transitive dependencies. It influences dependency resolution and ensures version consistency across different modules.
+- **Effect on Dependency Graph:** Platforms can **enforce** or **suggest versions** that are propagated through the dependency graph, affecting transitive dependencies and downstream consumers. They are a formal part of the dependency graph and can dictate the version chosen during resolution.
+
+[source,kotlin]
+.build.gradle.kts
+----
+plugins {
+    `java-platform`
+}
+
+dependencies {
+    constraints {
+        api("com.example:mylib:2.0.0") // This version will influence resolution
+    }
+}
+----
+
+== Version catalog vs. platform
```

Comment:
❗ Don't say vs., as you can use both in the same project - this is mentioned below.  I think both these sections (this and "Using a catalog with a platform") should be combined into just "Using catalogs with platforms")